packages/react-components/src/components/styles/RichTextEditor.styles.ts

```diff
@@ -257,7 +257,7 @@ export const editBoxRichTextEditorStyle = (): RichTextEditorStyleProps => {
 export const sendBoxRichTextEditorStyle = (isExpanded: boolean): RichTextEditorStyleProps => {
   return {
     minHeight: isExpanded ? '5rem' : '1.25rem',
-    maxHeight: '5rem'
+    maxHeight: '8rem'
```

Comment:
Previeously, `maxHeight` and `minHeight` were the same when sendBox is expanded. Just confirming that we don't want this behaviour for send box any more

Reply:
Referring to the current Sendbox, I think we would want the minHeight to be 5rem and maxHeight to be 8rem when expanded. However, I'll double check with Alex when he's back.